include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -0,0 +1,497 @@
+    //size_t                minMemoryMapAlignment;
+    uint32_t bufferViewAlignment;
+    uint32_t UBOAlignment;
+    uint32_t SSBOAlignment;
+    //int32_t               minTexelOffset;
+    //uint32_t              maxTexelOffset;
+    //int32_t               minTexelGatherOffset;
+    //uint32_t              maxTexelGatherOffset;
+    //float                 minInterpolationOffset;
+    //float                 maxInterpolationOffset;
+    //uint32_t              subPixelInterpolationOffsetBits;
+    //uint32_t              maxFramebufferWidth;
+    //uint32_t              maxFramebufferHeight;
+    //uint32_t              maxFramebufferLayers;
+    //VkSampleCountFlags    framebufferColorSampleCounts;
+    //VkSampleCountFlags    framebufferDepthSampleCounts;
+    //VkSampleCountFlags    framebufferStencilSampleCounts;
+    //VkSampleCountFlags    framebufferNoAttachmentsSampleCounts;
+    //uint32_t              maxColorAttachments;
+    //VkSampleCountFlags    sampledImageColorSampleCounts;
+    //VkSampleCountFlags    sampledImageIntegerSampleCounts;
+    //VkSampleCountFlags    sampledImageDepthSampleCounts;
+    //VkSampleCountFlags    sampledImageStencilSampleCounts;
+    //VkSampleCountFlags    storageImageSampleCounts;
+    //uint32_t              maxSampleMaskWords;
+    //VkBool32              timestampComputeAndGraphics;
+    float    timestampPeriodInNanoSeconds; // timestampPeriod is the number of nanoseconds required for a timestamp query to be incremented by 1 (a float because vulkan reports), use core::rational in the future
+    //uint32_t              maxClipDistances;
+    //uint32_t              maxCullDistances;
+    //uint32_t              maxCombinedClipAndCullDistances;
+    //uint32_t              discreteQueuePriorities;
+    float pointSizeRange[2];
+    float lineWidthRange[2];
+    //float                 pointSizeGranularity;
+    //float                 lineWidthGranularity;
+    //VkBool32              strictLines;
+    //VkBool32              standardSampleLocations;
+    //VkDeviceSize          optimalBufferCopyOffsetAlignment;
+    //VkDeviceSize          optimalBufferCopyRowPitchAlignment;
+    uint64_t nonCoherentAtomSize;
+            
+    /* Vulkan 1.1 Core  */
+    uint32_t subgroupSize;
+    core::bitflag<asset::IShader::E_SHADER_STAGE> subgroupOpsShaderStages;
+    bool shaderSubgroupBasic = false;
+    bool shaderSubgroupVote = false;
+    bool shaderSubgroupArithmetic = false;
+    bool shaderSubgroupBallot = false;
+    bool shaderSubgroupShuffle = false;
+    bool shaderSubgroupShuffleRelative = false;
+    bool shaderSubgroupClustered = false;
+    bool shaderSubgroupQuad = false;
+    bool shaderSubgroupQuadAllStages = false; //quadOperationsInAllStages;
+    //VkPointClippingBehavior    pointClippingBehavior;
+    //uint32_t                   maxMultiviewViewCount;
+    //uint32_t                   maxMultiviewInstanceIndex;
+    //VkBool32                   protectedNoFault;
```

Comment:
Related to [protected memory](https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.html#memory-protected-memory) 
we don't expose this